src/components/accounts/SetupSeedPhraseVerify.js

```diff
@@ -53,7 +54,7 @@ const SetupSeedPhraseVerify = ({
         <FormButton
             type='submit'
             color='blue'
-            disabled={!enterWord}
+            disabled={enterWord ? (globalAlert && globalAlert.success) : true}
```

Comment:
hmm, why is this change needed?

Reply:
thanks to that VERIFY button will stay disabled when addAccessKeySeedPhrase action is complete and it's waiting for redirection.

Reply:
>  it's waiting for redirection.

we shouldn't be waiting for redirection, we should just redirect immediately and then show alert on dashboard

Reply:
ok, by dashboard you mean dashboard or login? In this case, we would want to rebuild redirectToApp so it will do `history.push` instead of `window.location` for no redirect_url param case, so it will be possible to show Global Alert info from redux after redirection.

Also, we should handle `CLEAR_ALERT` action so the alert will be cleared from redux. We can simply clear Global Alert on every page change (for example in `componentDidMount` in `Routing` component), or we can clear it on `Dashboard` and `Login` components unmount. Or would you like to handle it another way?

Btw I'm not sure if `redirectToApp` is working correctly in one case, for example, if you start wallet with `/send-money` URL and create an account, it will redirect you to `/send-money` again (instead of dashboard). If we would like it to redirect to dashboard we should change a bit this condition:
https://github.com/nearprotocol/near-wallet/blob/c34b395f313484d46c5177b4d26dd2c95d85a299/src/actions/account.js#L100
or is that intentional?

Reply:
> ok, by dashboard you mean dashboard or login? In this case, we would want to rebuild redirectToApp so it will do history.push instead of window.location for no redirect_url param case, so it will be possible to show Global Alert info from redux after redirection.

Let's do this.

> Btw I'm not sure if redirectToApp is working correctly in one case, for example, if you start wallet with /send-money URL and create an account, it will redirect you to /send-money again (instead of dashboard). If we would like it to redirect to dashboard we should change a bit this condition:

This is intentional. We want to redirect the user to their original destination (/send-money in this example) after they complete the login flow.

Reply:
> ok, by dashboard you mean dashboard or login? 

I mean `redirectToApp`, i.e. either next URL or dashboard if there is no next URL. 

> so it will do history.push instead of window.location for no redirect_url param case, so it will be possible to show Global Alert info from redux after redirection.

sure, whatever works

> We can simply clear Global Alert on every page change (for example in componentDidMount in Routing component)

yes, it's desired to clear alert automatically on every page change

Reply:
@kcole16 I think there is unintentional consequence:
https://github.com/nearprotocol/near-wallet/issues/276

/cc @Patrick1904